pages/instructorEffectiveUser/instructorEffectiveUser.js

```diff
@@ -32,8 +33,21 @@ router.post('/', function(req, res, next) {
         res.clearCookie('pl_requested_mode');
         res.redirect(req.originalUrl);
     } else if (req.body.__action == 'changeUid') {
-        res.cookie('pl_requested_uid', req.body.pl_requested_uid, {maxAge: 60 * 60 * 1000});
-        res.redirect(req.originalUrl);
+        if (config.devMode) {
+            // Add the user in devmode to test access rules
```

Comment:
May want to be more explicit here - for example, make clear that nothing will happen if the user already exists.